.travis.yml

```diff
@@ -11,9 +12,8 @@ env:
 services:
   - memcached
 before_install:
-  - sudo apt-get install swig
-  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.deb && sudo dpkg -i elasticsearch-1.3.2.deb
-  - sudo /usr/share/elasticsearch/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
+  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.tar.gz && tar xvf elasticsearch-1.3.2.tar.gz
+  - elasticsearch-1.3.2/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
 install:
   - pip install tox
 before_script:
```

Comment:
if you add this you'll only have to download packages once:
````
   cache:
       directories:
         - /tmp/pip-cache
````

I also wrote a script to bundle elasticsearch @ https://github.com/mozilla/zamboni/pull/2821